docs/ensnode.io/src/components/overrides/ThemeSelect.astro

```
@@ -0,0 +1,2 @@
+---
```

My understanding is that creating this file is what turns off the light / dark mode picker? Is that right?

Could we comment the purpose of this file inside of this file? Appreciate your advice.

Also, what happens if a user's browser is configured by default to prefer dark mode? Will dark mode still be activated by default, even though we hide the theme picker at a UI level?

---

Good point. I've added the `ThemeProvider` override so it's light mode only.

The component overrides "override" the default behaviour yeah. Empty `ThemeSelect` means there is no theme select now.